### DISCUSSION TITLE
configure ldap with kubernetes secrets and assign policy to users

### DISCUSSION THREAD

**Discussion by r0b1ndot** [minio/minio#14890] [Q&A] [answered]
Hi,
can i provide a kubernetes-secret for my LDAP bind-pw instead using: `MINIO_IDENTITY_LDAP_LOOKUP_BIND_PASSWORD` ?

https://github.com/minio/minio/blob/master/docs/sts/ldap.md#configuring-adldap-on-minio

what i also want to archive is assign the policy to user by using the chart.
i know via the mc command line this is possible:
`mc admin policy set myminio mypolicy user='uid=james,cn=accounts,dc=myldapserver,dc=com'`
but i found no way doing this directly via the chart.

is this feature planned?

also use existing kubernetes secrets for the ldap-password would be highly appreciated.

**Comment by harshavardhana** [comment, marked as answer]
You need to do after the deployment..

**Reply by r0b1ndot** [reply]
@harshavardhana thanks, but for the Initial Setup, using a Secret for the LDAP bind-password would be highly appreciated since that information should be handled as Secret instead providing a plain password
